atomic_reactor/plugins/generate_sbom.py

```diff
@@ -285,6 +289,37 @@ def get_unique_and_sorted_components(
 
         return unique_components
 
+    def push_sboms_to_registry(self):
+        docker_config = os.path.join(self.workflow.conf.registries_cfg_path, '.dockerconfigjson')
+
+        if not os.path.exists(docker_config):
+            self.log.warning("Dockerconfig json doesn't exist in : '%s'",  docker_config)
+            return
+
+        tmpdir = tempfile.mkdtemp()
+        os.environ["DOCKER_CONFIG"] = tmpdir
+        sbom_type = '--type=cyclonedx'
+        dest_config = os.path.join(tmpdir, 'config.json')
+        os.symlink(docker_config, dest_config)
```

Comment:
why does the config need to be symlinked?

Reply:
because it has to be named 'config.json'

Reply:
would you mind adding that as a comment?

Reply:
added